deploy/template.yaml

```diff
@@ -421,6 +421,56 @@ Resources:
       FunctionName: !Ref AuthorizationFunction
       Principal: apigateway.amazonaws.com
 
+  DocumentSelectionFunction:
+    Type: AWS::Serverless::Function
+    DependsOn: TokenFunctionLogGroup
```

Comment:
should it be `DocumentSelectionLogGroup`?

Reply:
Fixed 🤦‍♂️